syntax = "proto3";

import "Error.proto";

message GetBawuInfoResIdl {
    message DataRes {
        message BawuTeam {
            int32 total_num = 1;
            message BawuRoleDes {
                string role_name = 1;
                message BawuRoleInfoPub {
                    int64 user_id = 2;
                    int32 role_id = 3;
                    string role_name = 4;
                    string portrait = 5;
                    int32 user_level = 6;
                    string user_name = 8;
                    string name_show = 9;
                }
                repeated BawuRoleInfoPub role_info = 2;
            }
            repeated BawuRoleDes bawu_team_list = 2;
        }
        BawuTeam bawu_team_info = 1;
    }
    DataRes data = 1;
    Error error = 2;
}